EDEngineer/Views/Popups/Graphics/GraphicSettings.cs

```diff
@@ -55,11 +55,6 @@ public double OpacityForEdit
             get => opacity;
             set
             {
-                if (Math.Abs(opacity - value) < 1)
```

Comment:
I think that was required to avoid some wpf crashes if I remember correctly - why are we removing this now?

Reply:
It was locking the opacity (issue #574)